### PR TITLE
Documentation update.

### PR DESCRIPTION
**Pull request by mmaker**
Some minor changes to readmes. 
I'm not sure those DHCP and NET env variables are actually superficial; but they do not appear in my Makefiles (`--xen` and `--unix`), and I was successfully able to run the `--unix` unikernel without them.

### CODE REVIEW

File: static_website/README.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
 On Unix, do:
 
 ```
-$ env NET=socket mirage configure --unix
-$ make depend
+$ mirage configure --unix
```
Comment (Drup) [inline static_website/README.md]: You should add `--net socket`.

Comment (samoht) [inline static_website/README.md]: this should be `mirage configure --net=socket --unix` with mirage 2.7.0

File: static_website/README.md [hunk 2]
```diff
@@ -14,9 +13,9 @@ content from htdocs directory.
 For a Xen DHCP kernel, do:
 
 ```
-$ env DHCP=true mirage configure --xen
+$ mirage configure --xen
```
Comment (Drup) [inline static_website/README.md]: You should add `--dhcp true`.

**Comment by Drup** [comment]
The reason it runs fine without them is that it happens to be the default value.

**Comment by samoht** [comment]
Thanks for the update! mirage.2.7.0 removed the env magic so now every configuration magic can go on the command-line directly.

**Comment by mmaker** [comment]
@Drup: mm, would you like me to add the command-line argument or it's okay to omit them since they're default?

**Comment by Drup** [comment]
I would tend to add them. @samoht ?

**Comment by samoht** [comment]
I think it's better to be explicit for these kind of examples, so yes you could add them :-) (and I prefer `--dhcp=true` instead of using a space). But nothing critical, so I'm happy to merge as is!

**Comment by mmaker** [comment]
should be okay now

**Comment by samoht** [comment]
Perfect, thanks a lot!

**Pull request merged**